src/cmap/commands.ts

```diff
@@ -6,6 +6,7 @@ import type { Long, Document, BSONSerializeOptions } from '../bson';
 import type { ClientSession } from '../sessions';
 import type { CommandOptions } from './connection';
 import { MongoRuntimeError, MongoInvalidArgumentError } from '../error';
+import { deserialize } from 'bson';
```

Comment:
You have to import BSON things from `src/bson.ts` to continue supporting our optional dependency logic.

Reply:
ok yeah good catch.  This was to make Sinon work, but I had to do this because I wasn't importing from `src/bson`.